easystacks/software.eessi.io/2023.06/accel/nvidia/eessi-2023.06-eb-4.9.4-2023a-CUDA.yml

```diff
@@ -0,0 +1,2 @@
+easyconfigs:
+  - cuDNN-8.9.2.26-CUDA-12.1.1.eb
```

Comment:
Made an attempt to arrange for that (only having one file ... or better not duplicates with the same content) in https://github.com/EESSI/software-layer/pull/772/commits/8e7a0e80ab3cefa1f3c0c2b7fbce71f2206f6102

Reply:
After todays GPU tiger meeting, we agreed to separate these after all. The list of _what is needed for `host_injections`_ is different from what we install. It's only those modules that we have to strip down that we have to then install in `host_injections`. Keeping that in a separate file is ok: it's not duplication, since _that_ is the unique list of package that are _needed_ in host injections (for nvidia GPU support)